content/10-managing-databases/04-database-replication/01-database-replication-introduction.mdx

```diff
@@ -0,0 +1,76 @@
+
+* improve the availability of data
+* increase the speed of data access
+* enhance server performance
+* accomplish preparation for disaster recovery
+* improve analytics
+
+### Improve the availability of data
+
+*Replication* improves the availability of data because it ensures against relying on a single point being failing and losing database access. If there were some sort of technical glitch, power outage, or any number of different occurrences that affected that single point, then your application would not be able to access the data needed for its proper function. 
+
+Data replication enhances the resiliency and reliability of systens by storing data across multiple nodes. If one node goes down, there is another one to step up with the exact same data on it to keep operations running smoothly.
+
+### Increase the speed of data access
+
+For companies both large and small, there may be employees located globally. Database replication helps ensure that for any given employee in the company that needs particular data that they can get it timely and without too much delay.
+
+Replicating data to local servers in the locations near to where it is being accessed, provides users with faster data access and queries. If a company had a single node in the USA, then an employee trying to pull data in Europe could experience latency when trying to access it. A company with a cluster in the USA as well as a replicated cluster in Europe would ensure that this employee can work efficiently with the data that they need when they need it.
+
+### Enhance server performance
+
+Database replication also reduces the load on the primary server of the database. If data was only able to be accessed on a single server, that server would be handling every request coming to the database. With replication, that load can be dispersed among other nodes in the distributed system. Because all of the nodes have identical replicas of the data, some queries could get handled by secondary nodes and leave the primary to handle only certain reads or only writes.
+
+Saving the primary for handling write operations can prove to be beneficial because if it were handling all reads and writes that may put immense strain on the processing power. This could make sense for a situation where there are not many operations coming in to the database. However, in times of scaling and growth, it is best practice to delegate tasks across a distributed system rather than putting all the load on a single node's plate.
+
+### Accomplish preparation for disaster recovery
+
+In the case of catastrophe, replication also secures recovery of what gets lost or corrupted. It facilitates recovery by maintaining accurate backups at well-monitored locations. If one node in the distribution goes down for any reason, there are nodes at the ready to help recover what was lost because of their identical replicas.
```

Comment:
```suggestion
In the case of catastrophe, replication also gives you the tools you to recover data that may have otherwise been lost or corrupted. It facilitates recovery by maintaining accurate backups at well-monitored locations. If one node in the distribution goes down for any reason, there are nodes at the ready to help recover what was lost because of their identical replicas.
```